frontend/freebase2wikidata.js

```diff
@@ -260,7 +260,56 @@ $(function() {
     var title = mw.config.get('wgTitle');
     return qidRegEx.test(title) ? title : false;
   }
+  
+ function generateNav() {
```

Comment:
As you can see there is little space for a "back to top" button, I can either place another link, as shown in the pic, or put a little arrow icon as a button. I would go with the former which is easier to understand.
  
![image](https://cloud.githubusercontent.com/assets/1914074/24323650/16c4a66a-1179-11e7-8764-29edd0c1e613.png)